Gemfile

```diff
@@ -26,17 +26,8 @@ gem "truncate_html"
 gem "rake", "~> 13.0"
 
 # Frontend
-gem "bourbon", "~> 7.0.0"
-gem "sass", "~> 3.4"
-gem "flight-for-rails"
-gem "i18n-js", ">= 3.0.0.rc11"
-gem "jquery-rails"
-gem "sassc"
-gem "therubyracer"
-gem "turbolinks"
-gem "uglifier", ">= 1.3.0"
-gem "chroma"
```

Comment:
I think chroma is needed (see the error in citizien charts module)

The rest of tems might not be needed as they are now depedencies in npm (flight, jquery, turbolinks...)

Reply:
I dug into Chroma gem and I've found it only in this file:
https://github.com/PopulateTools/gobierto/blob/4de026646f6d55153c4ba10a7b2fa82bb7e34543/app/views/gobierto_citizens_charters/charters/_single_charter_stylesheets_hook.html.erb#L9-L18

Looking at what it does, it seems only applies some opacity to an hex-color inside a CSS snippet, does it worth it keeping the gem? Perhaps it'd be easy replace it with no deps
cc/ @entantoencuanto

Reply:
code updated to remove this gem, so perhaps some test fail if they're testing Chroma somehow